src/gap-skipper.js

```diff
@@ -0,0 +1,86 @@
+import Ranges from './ranges';
+import videojs from 'video.js';
+
+export default class gapSkipper extends videojs.EventTarget {
```

Comment:
we don't have to extend EventTarget as we are not emitting any of our own events